enhancements/multi-arch/multiarch-manager-operator.md

```diff
@@ -333,14 +335,12 @@ type PodPlacementConfigSpec struct {
 	// +kubebuilder:default=Normal
 	LogVerbosity LogVerbosityLevel `json:"logVerbosity,omitempty"`
 
-	// ExcludedNamespaces is the list of namespaces the pod placement controller should ignore.
-	// When omitted, this means no opinion and the operator is left to choose a reasonable default.
-	// The current default behaviour is to ignore only reserved namespaces.
-	// Reserved namespaces are those matching the following globs; openshift-*,kube-*,hypershift-*.
-	// The list of reserved namespaces will always be excluded, independent of the value of this field.
-	// The list of reserved namespaces is subject to change over time.
-	// +kubebuilder:default:={openshift-*,kube-*,hypershift-*}
-	ExcludedNamespaces []string `json:"excludedNamespaces,omitempty"`
+	// NamespaceSelector decides whether to run the admission control policy on an object based
+	// on whether the namespace for that object matches the selector. If the
+	// object itself is a namespace, the matching is performed on
+	// object.metadata.labels. If the object is another cluster scoped resource,
+	// it never skips the policy.
```

Comment:
Was this copied from somewhere? I'm not sure this is the correct godoc/context for the particular use case of the selector

Reply:
this was the default namespace selector text that I lazily copied. I'll change this to reflect this specific case.

Reply:
Fixed too.

Reply:
```
// By default, the labelSelector consists of a single match expression that filters out
// pods in namespaces whose the label `multiarch.openshift.io/excludeNamespace` exists.
```

@aleskandro the `multiarch.openshift.io/excludeNamespace` is just an example here. I'm not sure if we should mention it in the docs. The administrator will have the flexibility to control it. It need not necessarily be an exclusion criteria.

Reply:
I removed it from the godoc